<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>673</width>
         <height>534</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFolder"/>
   <addaction name="actionAddFiles"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="actionScanItems"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomRealSize"/>
   <addaction name="actionZoomFitW"/>
   <addaction name="actionZoomFitH"/>
   <addaction name="actionDockFloat"/>
   <addaction name="actionDockVisible"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeFiles">
       <column>
        <property name="text">
         <string>图片文件</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片节点</string>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>分组2</string>
        </property>
        <item>
         <property name="text">
          <string>图片2</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/AddDir.ico</normaloff>:/ico/ico/AddDir.ico</iconset>
   </property>
   <property name="text">
    <string>AddFolder</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/AddFile.ico</normaloff>:/ico/ico/AddFile.ico</iconset>
   </property>
   <property name="text">
    <string>AddFiles</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/BlowUp.ico</normaloff>:/ico/ico/BlowUp.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/BlowDown.ico</normaloff>:/ico/ico/BlowDown.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionZoomRealSize">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/ActualSize.ico</normaloff>:/ico/ico/ActualSize.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomRealSize</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actionZoomFitW">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/FitWidth.ico</normaloff>:/ico/ico/FitWidth.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomFitW</string>
   </property>
   <property name="toolTip">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/DeletedIcon.ico</normaloff>:/ico/ico/DeletedIcon.ico</iconset>
   </property>
   <property name="text">
    <string>DeleteItem</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Exit.ico</normaloff>:/ico/ico/Exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionZoomFitH">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/SuitableHeight.ico</normaloff>:/ico/ico/SuitableHeight.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomFitH</string>
   </property>
   <property name="toolTip">
    <string>适合高度</string>
   </property>
  </action>
  <action name="actionScanItems">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/TraversalNode.ico</normaloff>:/ico/ico/TraversalNode.ico</iconset>
   </property>
   <property name="text">
    <string>ScanItems</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actionDockVisible">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/ActualSize.ico</normaloff>:/ico/ico/ActualSize.ico</iconset>
   </property>
   <property name="text">
    <string>DockVisible</string>
   </property>
   <property name="toolTip">
    <string>窗口可见</string>
   </property>
  </action>
  <action name="actionDockFloat">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/FormFloat.ico</normaloff>:/ico/ico/FormFloat.ico</iconset>
   </property>
   <property name="text">
    <string>DockFloat</string>
   </property>
   <property name="toolTip">
    <string>窗口浮动</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
